tests/test_services.py

```diff
@@ -483,6 +483,24 @@ def test_scoped_services():
     assert b is not d
 
 
+def test_scopeed_service_from_scoped_services():

```

Comment:
Please fix the typo (`scopeed` instead of `scoped`).

Reply:
Fixed and ran `make format`